frontend/src/container/WorkspaceContent.jsx

```diff
@@ -421,6 +421,10 @@ class WorkspaceContent extends React.Component {
       folder_open: newUrlSearchList.join(',')
     }
 
+    if (e && e.ctrlKey) {
+      window.open(PAGE.WORKSPACE.CONTENT_LIST(state.workspaceIdInUrl) + '?' + qs.stringify(newUrlSearchObject, { encode: false }), '_blank')
```

Comment:
this line is a bit too complicated, split the url concatenation in a const
```suggestion
      const url = PAGE.WORKSPACE.CONTENT_LIST(state.workspaceIdInUrl) + '?' + qs.stringify(newUrlSearchObject, { encode: false })
      window.open(url, '_blank')
```